SoundCloud Track Artwork fix

drac-dot-gay opened the pull request:
Changed track property that ArtworkService uses for soundcloud, it shouldn't throw a 404 now. There should probably also be a try/catch block to return null for the artwork instead of letting it bubble up, something which I can also quickly do.

closes #107

drac-dot-gay committed:
update artworkservice

it should now use the correct track property for soundcloud

angelobreuer commented:
If you want you can do the try-catch too, when we're at it. Thank you very much!

drac-dot-gay committed:
polishing and small changes

- added formal null check (just in case)
- added try/catch block just in case this happens again
